lib/src/bindings/5.0/generated_bindings.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: type=lint, unused_element, unused_field
+// ignore_for_file: type=lint, unused_element, unused_field, subtype_of_base_or_final_is_not_base_final_or_sealed
```

Comment:
Shouldn't the original `ffigen.yaml` (for Tizen 7.0 and older) be updated if this is necessary?

Reply:
I added ffigen.yaml files